techeria_controller/templates/smartphone.html

```diff
@@ -0,0 +1,205 @@
+  <!-- Custom styles for this template -->
+  <link href=" {% static 'css/style.css' %} " rel="stylesheet" />
+  <!-- responsive style -->
+  <link href=" {% static 'css/responsive.css' %} " rel="stylesheet" />
+</head>
+
+<body class="sub_page">
+  <div class="hero_area">
+    <!-- header section strats -->
+    <header class="header_section">
+      <div class="container-fluid">
+        <nav class="navbar navbar-expand-lg custom_nav-container pt-3">
+          <a class="navbar-brand" href=" {% static 'index.html' %} ">
+            <span>
+              Smartphones
+            </span>
+          </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <div class="d-flex ml-auto flex-column flex-lg-row align-items-center">
+              <ul class="navbar-nav  ">
+                <li class="nav-item active">
+                    <a class="nav-link" href=" {% url 'index' %}  ">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% static 'about.html' %} "> About </a>
+                </li>
+
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% static 'contact.html' %} ">Contact us</a>
+                </li>
+              </ul>
+              <form class="form-inline my-2 my-lg-0 ml-0 ml-lg-4 mb-3 mb-lg-0">
+                <button class="btn  my-2 my-sm-0 nav_search-btn" type="submit"></button>
+              </form>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <!-- end header section -->
+
+  </div>
+  <div class="bg">
+
+    <h3 class="my-4 border-bottom pb-1"><span class="text-muted">Search Results for </span>{{request.GET.q}}</h3>
+      <div class="container bcontent">
+          {% for q in smartphone%}
+          <hr />
+          <div class="card" style="width: 500px;">
+              <div class="row no-gutters">
+                  <div class="col-sm-5">
+                      <img class="card-img-top" src="/media/smartphones/{{q.image}}" alt="Card image cap">
+                  </div>
+                  <div class="col-sm-7">
+                      <div class="card-body">
+                          <h5 class="card-title">{{ q.name }}</h5>
+                          <p class="card-text">${{ q.price }}</p>
+                          <a href="#" class="btn btn-primary">Details</a>
+                      </div>
+                  </div>
+              </div>
+          </div>
+          {% endfor %}
+      </div>
+
+
+
+
+
+<!-- info section -->
+<section class="info_section layout_padding">
```

Comment:
This section should be deleted because it doesn't have any relevant information to our Techeria website, it only shows Lorem Ipsum text.

Reply:
yeah, i was gonna delete it!